ffmpeg/api_test.go

```diff
@@ -1288,6 +1288,42 @@ func TestTranscoder_OutputFPS(t *testing.T) {
 	outputFPS(t, Software)
 }
 
+func TestTranscoderAPI_ClipInvalidConfig(t *testing.T) {
+	// Test the following file open results on input: fail, success, fail, success
```

Comment:
Are we missing the success test cases here?

Also given clipping touches our timestamp logic, which is different for CPU and GPU, it might be a good idea to have this test case inside a common `clipConfig()` function that is called both for CPU here, and for GPU devices in `nvidia_test.go`.

Reply:
> Are we missing the success test cases here?

It was wrong comment, removed.

> , which is different for CPU and GPU

Can you elaborate on that?

Reply:
> Can you elaborate on that?

My bad, the timestamps should match between CPU and GPU pipelines. The difference only occurs if FPS filter (and our timestamp logic around it) is used. So we can keep a single test case here.